pkg/k8shandler/configuration_tmpl.go

```diff
@@ -26,26 +22,11 @@ gateway:
   expected_nodes: {{.RecoverExpectedShards}}
   recover_after_time: ${RECOVER_AFTER_TIME}
 
-io.fabric8.elasticsearch.kibana.mapping.app: /usr/share/elasticsearch/index_patterns/com.redhat.viaq-openshift.index-pattern.json
-io.fabric8.elasticsearch.kibana.mapping.ops: /usr/share/elasticsearch/index_patterns/com.redhat.viaq-openshift.index-pattern.json
-io.fabric8.elasticsearch.kibana.mapping.empty: /usr/share/elasticsearch/index_patterns/com.redhat.viaq-openshift.index-pattern.json
-
-openshift.config:
-  use_common_data_model: true
-  project_index_prefix: "project"
-  time_field_name: "@timestamp"
-
-openshift.searchguard:
-  keystore.path: /etc/elasticsearch/secret/admin.jks
-  truststore.path: /etc/elasticsearch/secret/searchguard.truststore
-
-openshift.kibana.index.mode: {{.KibanaIndexMode}}
-
 path:
   data: /elasticsearch/persistent/${CLUSTER_NAME}/data
   logs: /elasticsearch/persistent/${CLUSTER_NAME}/logs
 
-searchguard:
+opendistro_security:
   authcz.admin_dn:
   - CN=system.admin,OU=OpenShift,O=Logging
   config_index_name: ".searchguard"
```

Comment:
i think we should also change this index name since, IIRC, the types may conflict upon start up

Reply:
> we should also change this index name

What do you suggest?  @jcantrill - what are you using in the es proxy?

Reply:
I believe OD defaults to the use of `.security` which is reasonable to me

Reply:
Now using `.security` - not sure where else this needs to change - a quick search `git grep '[.]searchguard'` under origin-aggregated-logging shows several hits